### PR TITLE
Improve html

### PR DESCRIPTION
**Pull request by nono**
- Fix the glitch when the app is loading
- Use the favicon helper

### CODE REVIEW

**Comment by nono** [comment]
[ETU j'ai quelque fois un gros nuage au chargement de mes applications](https://trello.com/c/yRCkm1Dz/742-etu-jai-quelque-fois-un-gros-nuage-au-chargement-de-mes-applications)

File: src/targets/browser/index.ejs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="{{.Locale}}">
+<head>
```
Comment (gregorylegarec) [inline src/targets/browser/index.ejs]: m4dz point was that browsers are smart enought to fix this point by themself.

I a pretty much no opinion so this version fits me as well.

**Comment by gregorylegarec** [comment]
Thanks @nono 👍

File: src/targets/browser/index.ejs (hunk 1 above)
Comment (nono) [reply src/targets/browser/index.ejs]: I confirm: the browsers are smart enough to do that.

But this trick was not used consistently: `<head>` was still present in a lot of templates in our applications, and I prefer to be explicit. So, I've put again `<head>` in our templates, even if it is not strictly needed.

**Pull request merged**